commons/src/main/java/org/dhis2/commons/utils/OrgUnitHolder_2.java

```diff
@@ -0,0 +1,146 @@
+package org.dhis2.commons.utils;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.CheckBox;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.core.content.ContextCompat;
+
+import com.unnamed.b.atv.model.TreeNode;
+
+import org.dhis2.commons.R;
+import org.hisp.dhis.android.core.organisationunit.OrganisationUnit;
+
+import java.util.Locale;
+import java.util.Objects;
+
+import timber.log.Timber;
+
+/**
+ * QUADRAM. Created by ppajuelo on 19/10/2017.
+ */
+
+public class OrgUnitHolder_2 extends TreeNode.BaseNodeViewHolder<OrganisationUnit> {
```

Comment:
why is _2? Could not be ORgUnitHolder?

Reply:
Because already exist a file with tthis name (ORgUnitHolder)